src/app/pipes/shared/ksi-date.pipe.ts

```diff
@@ -0,0 +1,15 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'ksiDate'
+})
+export class KsiDatePipe implements PipeTransform {
+
+  transform(value: string, ...args: unknown[]): string {
+    const valueDate = new Date(value);
+    const offsetInHours = - valueDate.getTimezoneOffset() / 60;
+    const correctTime = valueDate.setHours(valueDate.getHours() + offsetInHours);
+    return new Date(correctTime).toLocaleString();
```

Comment:
Myslíš, že by to šlo upravit tak, že to to správné datum pošle spolu s předanými parametry originální date pipe? Díky tomu bychom mohli použít všechny možnosti Angulaří date pipe